crates/cheatnet/src/blockifier_structs.rs

```diff
@@ -0,0 +1,62 @@
+// structs copied from blockifier to make CallEntryPoint serializable
+use serde::{Deserialize, Serialize};
+use starknet_api::core::{ClassHash, ContractAddress, EntryPointSelector};
+use starknet_api::deprecated_contract_class::EntryPointType;
+use starknet_api::transaction::Calldata;
+
+#[derive(Clone, Debug, Default, Eq, PartialEq, Deserialize, Serialize)]
```

Comment:
Did you try alternative approaches instead of copying? Wouldn't manual serialization be better instead? Even as a makeshift solution

Reply:
I think we will just provide such structures in profiler as a library and forge will be able to use it to meet the standard. It will be much easier than realying on blockifier internals.